src/main/react/menu_app/src/components/Navbar.js

```diff
@@ -1,21 +1,21 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
```

Comment:
It feels like 0bb23474eb71959594bba52fec6cb4597ef7fb23 and f907dc0ab90aab93131ec905122b1bf89868d202 should be squashed now

At a glance is not clear, that the style is changed by switching the component. And it is even more confusing given you've done a swap from `NavLink` to `Link` before with no visible reason

Reply:
You sure you mean these two? I guess it should be [this one](https://github.com/KateLocate/menu_generator/pull/6/commits/5e14b003f4cac38a2f48de8a714f832e70a142e2) and [this one](https://github.com/KateLocate/menu_generator/commit/f907dc0ab90aab93131ec905122b1bf89868d202)